pkg/prometheus/promcfg_test.go

```diff
@@ -7647,3 +7647,89 @@ scrape_configs:
 		})
 	}
 }
+
+func TestStorageSettingMaxExemplars(t *testing.T) {
```

Comment:
could you add another test case with version < 2.29?

Reply:
done :)